### PR TITLE
New language swedish

### PR DESCRIPTION
**Pull request by felixdro**
## What does this PR do?
Adding Swedish translations to Coral 5 
New files: src/locales/sv/*, src/core/server/locales/sv/*
Updated files: locales.ts , general.spec.tsx.snap

## How do I test this PR?
Select Swedish

### CODE REVIEW

**Comment by losowsky** [comment]
Thanks for this, Felix! Is Coral live on computersweden/other IDG sites yet? We'd love to see it working in Swedish?

File: src/locales/sv/auth.ftl [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+addEmailAddress-addEmailAddressButton =
+  Add Email Address
+
+## Create Password
+createPassword-createPasswordHeader = Create Password
+createPassword-whatItIs =
+  To protect against unauthorized changes to your account,
+  we require users to create a password.
+createPassword-createPasswordButton =
+  Create Password
```
Comment (kgardnr) [inline src/locales/sv/auth.ftl]: Hey @felixdro are you able to translate this as well? If you want to do it later, just please remove this file - it always defaults to English if there are missing translations. Thank you!

File: src/locales/sv/install.ftl [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  scheme (ex. http://localhost:3000, https://staging.domain.com,
+  https://domain.com).
+install-permittedDomains-permittedDomains = Permitted Domains
+install-permittedDomains-permittedDomainsTextField =
+  .placeholder = Domains
+install-permittedDomains-permittedDomainsDescription = Insert domains separated by comma
+
+install-finalStep-description = Thanks for installing { -product-name }! We sent an email to verify your email address. While you finish setting up the account, you can start engaging with your readers now.
+install-finalStep-goToTheDocs = Go to the Docs
+install-finalStep-goToAdmin = Go to Admin
```
Comment (kgardnr) [inline src/locales/sv/install.ftl]: Hey @felixdro are you able to translate this as well? If you want to do it later, just please remove this file - it always defaults to English if there are missing translations. Thank you!

File: src/core/server/locales/sv/errors.ftl [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+error-integrationDisabled = Specified integration is disabled.
+error-passwordResetTokenExpired = Password reset link expired.
+error-emailConfirmTokenExpired = Email confirmation link expired.
+error-rateLimitExceeded = You're trying to do that too many times. Please wait and try again.
+error-inviteTokenExpired = Invite link has expired.
+error-inviteRequiresEmailAddresses = Please add an email address to send invitations.
+error-passwordIncorrect = Incorrect password. Please try again.
+error-usernameAlreadyUpdated = You may only change your username once every { framework-timeago-time }.
+error-persistedQueryNotFound = The persisted query with ID { $id } was not found.
+error-rawQueryNotAuthorized = You are not authorized to execute this query.
```
Comment (kgardnr) [inline src/core/server/locales/sv/errors.ftl]: Hey @felixdro are you able to translate this as well? If you want to do it later, just please remove this file - it always defaults to English if there are missing translations. Thank you!

**Review by kgardnr — CHANGES_REQUESTED** [review]

**Comment by felixdro** [comment]
> Thanks for this, Felix! Is Coral live on computersweden/other IDG sites yet? We'd love to see it working in Swedish?

And thanks for developing Coral! Yes it's live on several of our sites now! We recently released for out members and havn't got that many comments just yet. 

Here is an open article (with one comment currently) https://pcforalla.idg.se/2.1054/1.728097/bluescreen-view#commentStream

File: src/locales/sv/auth.ftl (hunk 1 above)
Comment (felixdro) [reply src/locales/sv/auth.ftl]: This file is now 100 % translated

File: src/locales/sv/install.ftl (hunk 2 above)
Comment (felixdro) [reply src/locales/sv/install.ftl]: This file is now partially translated

File: src/core/server/locales/sv/errors.ftl (hunk 3 above)
Comment (felixdro) [reply src/core/server/locales/sv/errors.ftl]: This file is now partially translated too.

File: src/locales/sv/install.ftl (hunk 2 above)
Comment (felixdro) [reply src/locales/sv/install.ftl]: @kgardnr Or is it better to remove the file from the Swedish translation instead of only partially translating it?

**Comment by kgardnr** [comment]
Hey @felixdro do you want me to leave this PR open so you can contribute the missing translations?

Also, I can fix those merge conflicts or you can, up to you!

File: src/core/server/locales/sv/errors.ftl (hunk 3 above)
Comment (felixdro) [reply src/core/server/locales/sv/errors.ftl]: This file is 100 % translated now

File: src/locales/sv/install.ftl (hunk 2 above)
Comment (felixdro) [reply src/locales/sv/install.ftl]: This file is deleted now

**Comment by felixdro** [comment]
> Hey @felixdro do you want me to leave this PR open so you can contribute the missing translations?
> 
> Also, I can fix those merge conflicts or you can, up to you!

@kgardnr I deleted the Swedish translation file src/locales/sv/install.ftl and translated the server/error.ftl 100% and pushed those changes.
There was a conflict with the new Finnish translation in two files that I had missed but I solved them too. (src/core/common/helpers/i18n/locales.ts and src/core/client/admin/test/configure/__snapshots__/general.spec.tsx.snap)

 ̶B̶u̶t̶ ̶n̶o̶w̶ ̶t̶h̶e̶ ̶t̶e̶s̶t̶s̶ ̶f̶a̶i̶l̶s̶ ̶a̶n̶d̶ ̶I̶ ̶d̶o̶n̶'̶t̶ ̶k̶n̶o̶w̶ ̶w̶h̶y̶ ̶(̶I̶'̶m̶ ̶a̶ ̶b̶i̶t̶ ̶n̶e̶w̶ ̶t̶o̶ ̶t̶h̶i̶s̶ ̶a̶n̶d̶ ̶h̶o̶p̶e̶ ̶I̶ ̶d̶i̶d̶n̶'̶t̶ ̶d̶o̶ ̶a̶n̶y̶t̶h̶i̶n̶g̶ ̶w̶r̶o̶n̶g̶)̶.̶ ̶T̶e̶l̶l̶ ̶m̶e̶ ̶i̶f̶ ̶I̶ ̶s̶h̶o̶u̶l̶d̶ ̶d̶o̶ ̶a̶n̶y̶t̶h̶i̶n̶g̶ ̶e̶l̶s̶e̶,̶ ̶i̶f̶ ̶t̶h̶e̶r̶e̶ ̶i̶s̶ ̶a̶n̶y̶ ̶m̶o̶r̶e̶ ̶m̶e̶r̶g̶e̶ ̶c̶o̶n̶f̶l̶i̶c̶t̶s̶ ̶I̶ ̶w̶o̶u̶l̶d̶ ̶b̶e̶ ̶g̶l̶a̶d̶ ̶i̶f̶ ̶y̶o̶u̶ ̶c̶a̶n̶ ̶f̶i̶x̶ ̶t̶h̶e̶m̶.̶ ̶T̶h̶a̶n̶k̶s̶!̶

Edit: I fixed the new errors and pushed them here again. Now all the tests pass.

**Review by kgardnr — APPROVED** [review]

**Pull request merged**